Libplanet/Store/BlockSet.cs

```diff
@@ -74,7 +81,30 @@ public override bool ContainsKey(HashDigest<SHA256> key)
 
         public override bool Remove(HashDigest<SHA256> key)
         {
-            return Store.DeleteBlock(key);
+            bool deleted = Store.DeleteBlock(key);
+
+            if (deleted)
+            {
+                _cache.Remove(key);
```

Comment:
Are there no any cases that the block with key does not exist on store, but on cache?

Reply:
I guess.. it's possible because we're exposing `IStore.DeleteBlock()` as public. (as far I know, we don't use that directly in Libplanet).

I'll add a containing check on `BlockPolicy<T>.GetBlock()` but it can degrade performance... 😕

Reply:
How about just remove the if statement?

Reply:
Did you mean https://github.com/planetarium/libplanet/pull/1013/files#r493167744 ?

Reply:
Yes right